pkgs/data/fonts/lucide/default.nix

```
@@ -0,0 +1,33 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchurl,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "lucide";
+  version = "0.541.0";
+
+  src = fetchurl {
+    url = "https://unpkg.com/lucide-static@${version}/font/Lucide.ttf";
+    sha256 = "sha256-H8ACnTkbR4N3yYbWuteZoqfPlLu+OGm/NEfweBad8uM=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/truetype
+    cp -a *.ttf $out/share/fonts/truetype/
+  '';
+
+  sourceRoot = ".";
+  unpackCmd = ''
+    ttfName=$(basename $(stripHash $curSrc))
+    cp $curSrc ./$ttfName
+  '';
+
+  meta = with lib; {
+    homepage = "https://lucide.dev/";
+    description = "Open-source icon library that provides 1000+ icons";
+    platforms = platforms.all;
+    license = licenses.isc;
+    maintainers = [ "janTatesa" ];
```

```suggestion
    maintainers = with lib.maintainers; [ janTatesa ];
```
Maintainers are attributes declared in [`maintainers/maintainer.list.nix`](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix), not strings.